frontend/src/CreateOrganizationPage.js

```diff
@@ -18,7 +17,7 @@ export default function CreateOrganizationPage() {
   const history = useHistory()
 
   const [infoState, changeInfoState] = useState({
-    isHidden: true,
+    isVisable: false,
```

Comment:
Just a minor typo, should be `isVisible`.

Reply:
I've searched and replaced every instance of visable with visible, so these all should be good now.

I do hate English spelling sometimes...